src/client/windowsworkstation/win10-azuredeploy.json

```diff
@@ -67,7 +67,7 @@
     "publicIPAddressName": "[concat('publicip-',variables('uniqueName'))]",
     "vmName": "[concat('vm-',variables('uniqueName'))]",
     "imageReference": {
-      "publisher": "microsoftvisualstudio",
+      "publisher": "MicrosoftVisualStudio",
```

Comment:
I don't think that has been generated in awhile. Same with the description change line 53

Reply:
waiting until [this pipeline](https://dev.azure.com/averevfxt/vfxt-github/_build/results?buildId=9677&view=results) passes